holmes/core/supabase_dal.py

```diff
@@ -173,9 +174,18 @@ def get_resource_instructions(self, type: str, name: str) -> List[str]:
             .execute()
         )
         if res.data:
-            return res.data[0].get("runbook").get("instructions")
+            instructions = res.data[0].get("runbook").get("instructions")
+            documents_data = res.data[0].get("runbook").get("documents")
+            documents = []
+            for document_data in documents_data:
```

Comment:
Will this fail if there are no documents? (documents_data will be None, so not iterable)?

Reply:
Yes good catch, this will fail. Fixed